cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts

```diff
@@ -9,7 +9,7 @@ context('Wallet - Transaction Authorization with Error', () => {
 
   before(function () {
     cy.createEmptyWallet(true)
-    cy.sendDFItoWallet().sendTokenToWallet(['BTC']).wait(10000)
+    cy.sendDFItoWallet().sendTokenToWallet(['BTC']).wait(5000)
```

Comment:
reduce wait time, DFI token transfer (without splitting into 2 actions in subsequent block) is automatically done in whale playground now.